src/iris/webhooks/grafana.py

```diff
@@ -1,19 +1,25 @@
 import datetime
+import time
 import logging
 import ujson
 from falcon import HTTP_201, HTTPBadRequest, HTTPInvalidParam
 
 from iris import db
 from iris import utils
+from iris.webhooks.webhook import webhook
+from iris.custom_incident_handler import CustomIncidentHandlerDispatcher
 
 logger = logging.getLogger(__name__)
 
 
-class grafana(object):
+class grafana(webhook):
     allow_read_no_auth = False
 
+    def __init__(self, config):
+        self.custom_incident_handler_dispatcher = CustomIncidentHandlerDispatcher(config)
```

Comment:
Do we need to define this constructor? It seems to be a duplicate of the inherited constructor from `webhook`, and the other web hook implementations that inherit from `webhook` are still constructor-less.